Dockerfile

```diff
@@ -3,42 +3,52 @@
 ARG goversion
```

Comment:
This looks like a lot of changes, but it's just because I renamed Dockerfile.scratch over the existing Dockerfile.